CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.2.1-nullsafety-dev
```

Comment:
I think you can stay on 2.2.0-nullsafety-dev.
The "-dev" suffix means (or should mean) that the changes haven't been published yet, so you're just including more into the same release.

If anyone is picking the version directly from github, they are using a hash, not a version, so they won't be affected.

Reply:
Done.